Additional packages needed to build version 1.6.0 on local machine with Ubuntu 22.04LTS

**Describe the bug**
 I'm following this tutorial (https://spack-stack.readthedocs.io/en/1.6.0/NewSiteConfigs.html#prerequisites-ubuntu-one-off) to build `spack` & template `unified-dev` on my local (os: Kubuntu 22.04 LTS, `6.5.0-21-generic #21~22.04.1-Ubuntu`, `gcc@11.4.0`). 

For the section "Install basic OS packages as root", bullet "#Misc", I need two more packages, `autopint` and `gettext`, otherwise, I encountered problems and failed to finish building stack.


**To Reproduce**
Follow the section 6.2.2. "Prerequisites: Ubuntu (one-off)" (https://spack-stack.readthedocs.io/en/1.6.0/NewSiteConfigs.html#prerequisites-ubuntu-one-off)"

**Expected behavior**
I got two errors
1. Problem related to `autopoint`:
```
Can't exec "autopoint": No such file or directory at /usr/share/autoconf/Autom4te/FileUtils.pm.
```

2. Problem related to `gettext`:
```
/bin/bash: line 1: xgettext: command not found
make[2]: *** [Makefile:674: cxpm.po] Error 127
make[2]: *** Waiting for unfinished jobs....
mv -f .deps/cxpm.Tpo .deps/cxpm.Po
make[2]: Leaving directory '/home/cda/fast/pkg/spack-stack/cache/build_stage/spack-stage-libxpm-3.5.17-psnzjhv7rkydtqscil7kgk3hsu27i4j2/spack-src/cxpm'
make[1]: *** [Makefile:491: all-recursive] Error 1
make[1]: Leaving directory '/home/cda/fast/pkg/spack-stack/cache/build_stage/spack-stage-libxpm-3.5.17-psnzjhv7rkydtqscil7kgk3hsu27i4j2/spack-src'
make: *** [Makefile:400: all] Error 2
```

**System:**
Local machine:
- os: Kubuntu 22.04 LTS, `6.5.0-21-generic #21~22.04.1-Ubuntu`
- `gcc@11.4.0`

**Solution**
I Need to install the additional packages
```
sudo apt-get install autopoint
sudo apt-get install gettext
```

**Consequence**
Successfully passed these previously-failed sections, and finished the installation

--- a/doc/source/NewSiteConfigs.rst
+++ b/doc/source/NewSiteConfigs.rst
@@ -409,6 +409,8 @@ The following instructions were used to prepare a basic Ubuntu 20.04 or 22.04 LT
    apt install -y bzip2
    apt install -y unzip
    apt install -y automake
+   apt install -y autopoint
+   apt install -y gettext
    apt install -y xterm
    apt install -y texlive
    apt install -y libcurl4-openssl-dev

